demo/main.js

```diff
@@ -66,7 +66,7 @@ let start;
 
 class Demo extends Component {
   setup() {
-    this.state = useState({ key: 0 });
+    this.state = useState({ key: 0, displayHeader: false });
```

Comment:
funnily enough, the issue this commit is fixing is illustrated when the header appears.
o-spreadsheet is bigger and it auto-size in a not-so-good manner